app/javascript/Components/table/table.jsx

```diff
@@ -47,17 +47,20 @@ export default function Table({
   renderSubComponent,
   getRowCanExpand,
   columnFilters: externalColumnFilters,
+  onColumnFiltersChange: externalOnColumnFiltersChange,
 }) {
-  const [columnFilters, setColumnFilters] = React.useState([]);
+  const [internalColumnFilters, setInternalColumnFilters] = React.useState([]);
   const [columnSizing, setColumnSizing] = React.useState({});
-  const [columnVisibility, setColumnVisibility] = React.useState({inactive: false});
+  const [columnVisibility, setColumnVisibility] = React.useState({hidden: false});
```

Comment:
In general renaming state values like this should be done carefully, as there is usually a reason for the name.

In this case, `"inactive"` is an id used in at least one other table that uses the new `Table` component (*search the codebase to find it*).

For now you can revert this change and instead modify the the column definition in the `Course SummariesTable`.